src/app/debugger/debugger.js

```diff
@@ -17,16 +17,14 @@ function Debugger (container, sourceHighlighter, localRegistry) {
   this._deps = {
     offsetToLineColumnConverter: this._components.registry.get('offsettolinecolumnconverter').api,
     editor: this._components.registry.get('editor').api,
-    compiler: this._components.registry.get('compiler').api
+    compiler: this._components.registry.get('compiler').api,
+    compilersArtefacts: this._components.registry.get('compilersartefacts').api
```

Comment:
depending on the order or merges, we need to either add this change to the debugger in other PR or to this one

Reply:
I think that s managed below: https://github.com/ethereum/remix-ide/pull/1536/files#diff-23075dc7d1a122a47f6be2a98ee4770cR27

in `compilationResult` instead of returning the solidity compiler result, we return the last one, could be solidity or other one